### PR TITLE
Make video a parameter

### LINKED ISSUE #41: Make video a parameter
Add parameter to remove assumption miniscene folder path is *|video_name

### PR DESCRIPTION
**Pull request by zhong-al**
Resolves #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,14 +103,15 @@ python -m pip install git+https://github.com/zhong-al/SlowFast@797a6f3ae81c49019
...
-python miniscene2behavior.py --config [config path] --checkpoint [checkpoint path] --gpu_num [number of gpus available] --miniscene [miniscene path] --output [output path]
+miniscene2behavior --config [config path] --checkpoint [checkpoint path] --gpu_num [number of gpus available] --miniscene [mini-scenes' folder path] --video [video name] --output [output path]
 ```
 
 **Notes:**
  - If building detectron2's wheel fails, loading a different gnu module may help (`module load gnu/11.2.0`)
  - If the config hasn't been extracted yet, the script will write it to `config`. 
  - `checkpoint` should be the path to `checkpoint_epoch_00075.pyth`. 
  - If `gpu_num` is 0, the model will use CPU. Using at least 1 GPU greatly increases inference speed. If you're using OSC, you can request a node with one GPU by running `sbatch -N 1 --gpus-per-node 1 -A [account] --time=[minutes] [bash script]`.
+ - mini-scenes are clipped videos focused on individual animals and video is the raw video file from which mini-scenes have been extracted
```
Comment (egrace479) [inline README.md:114]:
```suggestion
 - mini-scenes are clipped videos focused on individual animals and video is the raw video file from which mini-scenes have been extracted.
```

**Review by egrace479 — COMMENTED** [review]
Should this actually be directed at the branch from PR #40 ? It seems to include some, but not all of those changes.

File: README.md (hunk 1 above)
Comment (zhong-al) [reply README.md:114]: Should I commit this change or rebase on the branch from PR #40?

Comment (egrace479) [reply README.md:114]: I think this note is more relevant to this PR since it's where the video parameter gets added.

**Review by jennamk14 — APPROVED** [review]
LGTM

**Pull request merged**